blueocean-web/src/main/js/DevelopmentFooter.jsx

```diff
@@ -5,14 +5,6 @@ import revisionInfo from '../../../target/classes/io/jenkins/blueocean/revisionI
 
 export class DevelopmentFooter extends Component {
     render() {
-        if (!revisionInfo || !revisionInfo.name) {
-            var blueOceanVersion = document.getElementsByTagName('head')[0].getAttribute('data-blue-ocean-version');
-            return (
-                <div className="development-footer">
-                    <span>Blue Ocean UI v{blueOceanVersion}</span>
```

Comment:
But now we won't see the revision number on screen, right? From the ticket, am I reading right that there will be a follow-on to restore it?